pkg/assets/manifests.go

```diff
@@ -0,0 +1,44 @@
+package assets
+
+var BootstrapKubernetesAPIServerTemplate = []byte(`apiVersion: v1
```

Comment:
@deads2k @sttts i like gotemplates better than YAML files we have to substitute strings in or openshift templates, but this is up to discussion

Reply:
Yaml gotemplate files +1

Reply:
done.